### DISCUSSION TITLE
soil sampling

### DISCUSSION THREAD

**Discussion by SeregaSk1982** [Courseplay/courseplay#7542] [Q&A] [answered]
and when the opportunity will be added for the PRECISION FARMING plugin, so that you can take soil samples in the fields?

**Comment by Tensuko** [comment, marked as answer]
This is already possible with this:
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=195655&title=fs2019
and this:
https://www.farming-simulator.com/mod.php?lang=de&country=de&mod_id=210609&title=fs2019

The base one won't be added as it is too much work to make perfect patterns with that round area.

Unfold the tools, turn them on, let CP run in Mode6 and its fine.

**Reply by SeregaSk1982** [reply]
thanks
